obs_status.yml

```diff
@@ -0,0 +1,1995 @@
+  status: 1
+- agasc_id:
+  - 1200888464
+  comments: NSM at start of observation
+  mp_starcat_time: 2017:090:18:27:10.378
+  status: 1
+- agasc_id:
+  - 45616472
+  comments: Faint star affected by imposter. Set mag to MAXMAG
+  mp_starcat_time: 2017:119:02:36:19.023
+  status: 1
+- agasc_id:
+  - 311824376
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2017:198:09:20:24.644
+  status: 1
+- agasc_id:
+  - 913441888
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2017:259:02:11:04.769
+  status: 1
+- agasc_id:
+  - 377102496
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2017:314:20:46:46.000
+  status: 1
+- agasc_id:
+  - 92812824
+  comments: Almost never tracked. Set mag to MAXMAG
+  mp_starcat_time: 2017:354:04:35:24.949
+  status: 1
+- agasc_id:
+  - 653529952
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2018:002:23:37:48.551
+  status: 1
+- agasc_id:
+  - 123874048
+  comments: Bright star
+  mp_starcat_time: 2018:014:04:03:16.301
+  status: 1
+- agasc_id:
+  - 797847184
+  comments: Tracking affected by imposter. Set mag to 12.3 +- 0.2
+  mp_starcat_time: 2018:068:11:54:25.784
+  status: 1
+- agasc_id:
+  - 1105068512
+  comments: spoiler 22 arcsec away
+  mp_starcat_time: 2018:069:03:04:54.838
+  status: 1
+- agasc_id:
+  - 872547280
+  comments: Large mag variation. Star is a known eclipsing binary
+  mp_starcat_time: 2018:074:18:43:05.432
+  status: 1
+- agasc_id:
+  - 651037128
+  comments: Almost never acquired. Faint star affected by imposters. Marginally acceptable
+  mp_starcat_time: 2018:081:21:42:04.564
+  status: 0
+- agasc_id:
+  - 899950912
+  comments: Faint star. Affected by hot pixels. Set mag to MAXMAG
+  mp_starcat_time: 2018:155:22:59:34.754
+  status: 1
+- agasc_id:
+  - 738075032
+  comments: Spoiler 30 arcsec away
+  mp_starcat_time: 2018:158:10:17:36.762
+  status: 1
+- agasc_id:
+  - 49818760
+  comments: Faint star. Affected by imposter. Set mag to MAXMAG
+  mp_starcat_time: 2018:165:06:34:21.353
+  status: 1
+- agasc_id:
+  - 745014832
+  comments: Tracking affected by imposter
+  mp_starcat_time: 2018:198:01:56:54.706
+  status: 1
+- agasc_id:
+  - 974925328
+  comments: Faint star. Tracking affected by imposters. Marginally acceptable
+  mp_starcat_time: 2018:214:12:04:13.567
+  status: 0
+- agasc_id:
+  - 8395104
+  comments: Magnitude changes within observation. Eclipsing binary
+  mp_starcat_time: 2018:295:05:40:26.899
+  status: 1
+- agasc_id:
+  - 168038608
+  comments: Faint star. Tracking affected by imposters. Marginally acceptable
+  mp_starcat_time: 2018:330:13:00:19.280
+  status: 0
+- agasc_id:
+  - 350392600
+  comments: Bright star. Never acquired. Set mag to 5.0
+  mp_starcat_time: 2019:012:01:07:46.603
+  status: 1
+- agasc_id:
+  - 109061768
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2019:035:04:23:05.219
+  status: 1
+- agasc_id:
+  - 989727248
+  comments: UNKNOWN
```

Comment:
For me, that's a "don't care" because it has a bunch of reasonable observations.